src/Relation/Binary/Consequences.agda

```diff
@@ -44,6 +44,31 @@ module _ {_∼_ : Rel A ℓ} {P : Pred A p} where
   resp⇒¬-resp : Symmetric _∼_ → P Respects _∼_ → (∁ P) Respects _∼_
   resp⇒¬-resp sym resp x∼y ¬Px Py = ¬Px (resp (sym x∼y) Py)
 
+------------------------------------------------------------------------
+-- Proofs for negation
+
+module _ {_∼_ : Rel A ℓ} where
+  private
+    _≁_ : A → A → Set _
+    x ≁ y = ¬ (x ∼ y)
+
+  sym⇒≁-sym : Symmetric _∼_ → Symmetric _≁_
```

Comment:
Looking at the rest of the file, I think the most consistent names for these two properties would be `sym⇒¬-sym` and `comp⇒¬-trans`

Reply:
Done.